apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -157,6 +159,15 @@ model GuildActivityType {
   @@unique([guild_id, activity_type_id])
 }
 
+model GuildTwitterIntegration {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @default(now()) @updatedAt
+  guild_id  Int      @unique
+  Guild     Guild    @relation(fields: [guild_id], references: [id])
+  hashtag   String   @unique
```

Comment:
definitely great call to start with this unique, and we can always relax it a bit if we decide to